packages/agent-infra/mcp-servers/browser/package.json

```diff
@@ -49,14 +49,14 @@
   "dependencies": {
     "@agent-infra/logger": "workspace:*",
     "@agent-infra/browser-context": "0.1.2",
+    "@agent-infra/browser-use": "workspace:*",
```

Comment:
Not including it in deps but placing it in devDeps is to bundle it into dependencies, reducing runtime dependency installation.